tests/e2e/create/testdata/moduleconfig/invalid/icons-withouth-name.yaml

```diff
@@ -0,0 +1,7 @@
+name: kyma-project.io/module/template-operator
```

Comment:
please fix the typo in the file name "without"

Reply:
Thanks! Done.